Editor/TextProvider.cs

```diff
@@ -255,6 +255,7 @@ internal sealed class TextProvider
             { Strings.AnywherePageComputeTitle, "Add Your Compute to the Fleet"},
             { Strings.AnywherePageComputeNameLabel, "Compute name"},
             { Strings.AnywherePageComputeIPLabel, "IP address"},
+            { Strings.AnywherePageComputeIPDescription, "127.0.0.1"},
```

Comment:
This shouldn't be in TextProvider, it's not text to translate, but a value to provide to an API call.